tests/test_expence.py

```diff
@@ -0,0 +1,113 @@
+import json
+
+from django.utils import timezone
+from django.contrib.auth.models import User
+from rest_framework.test import APITestCase
+from django.db.models import Q
+
+from . import TomatoTest
+from wallet.serializers import ExpenseSerializer
+from wallet.views import ExpenseViewSet
+from wallet.models import Category, Expense
+
+
+class ExpenceTest(TomatoTest):
+    def setUp(self):
+        super().setUp()
+        self.no_owner_cate = Category.objects.create(name=self.cname)
```

Comment:
Nitpick: as we have pretty good auto-complete nowadays, I personally prefer to spell it all out. It reads much better later.